packages/modules/web-viewer-react/src/services/ItwinViewer.ts

```diff
@@ -17,7 +17,7 @@ import ReactDOM from "react-dom";
 import { ViewerFrontstage } from "..";
 import { Viewer } from "../components/Viewer";
 import {
-  ItwinViewerParams,
```

Comment:
I believe we can just delete iTwinViewerParams interface from the base viewer-react package. It is web-specific now. Looks like it was just left behind once I moved it to the web package. 

Once we do that, we can leave the name the same (iTwinViewerParams) but just export it from the web package now. It will no longer be a breaking change in the web and the breaking change in the base is unlikely to be used in desktop apps so we should be good there as well.

Reply:
So are we doing a major ver bump of the base-viewer pkg? And that will propagate to the web and desktop viewer pkgs as minor bump?

Reply:
I don't think we need a major bump anywhere. We don't advocate using the base package anywhere, and even if somebody did, they would not be using that interface. The only potential breakage is that this interface would no longer be exposed to the desktop. However, its only use case is for the js (CDN) web use case, so no issue there